server/lib/search.js

```diff
@@ -167,18 +168,30 @@ export const searchCollectivesInDB = async (
   }
 
   // Build the query
+
+  const sortSubqueries = {
+    ACTIVITY: `
+      SELECT COALESCE(SUM(ABS("netAmountInCollectiveCurrency")), 0)
```

Comment:
`COUNT` could be a better metric than `SUM` IMO. With 'SUM`, anyone adding a fake $1000000 through added funds will arrive at the top of the page directly. It's easier to fake amounts than the number of transactions.